ticdc/ticdc-faq.md

```diff
@@ -55,6 +55,111 @@ The expected output is as follows:
 >
 > This feature is introduced in TiCDC 4.0.3.
 
+## How to verify if TiCDC has replicated all updates after upstream stops updating?
+
+After the upstream TiDB cluster stops updating, you can verify if replication is complete by comparing the latest [TSO](/glossary.md#tso) timestamp of the upstream TiDB cluster with the replication progress in TiCDC. If the TiCDC replication progress timestamp is greater than or equal to the upstream TiDB cluster's TSO, then all updates have been replicated. To verify replication completeness, perform the following steps:
+
+1. Get the latest TSO timestamp from the upstream TiDB cluster.
+
+    > **Note:**
+    >
+    > Use the [`TIDB_CURRENT_TSO()`](/functions-and-operators/tidb-functions.md#tidb_current_tso) function to get the current TSO, instead of using functions like `NOW()` that return the current time.
```

Comment:
```suggestion
    > Use the `TIDB_CURRENT_TSO()` function to get the current TSO, instead of using functions like `NOW()` that return the current time.
```